CONTRIBUTING.md

```diff
@@ -42,6 +40,6 @@ public class LongClassName {
 }
 ```
 
+If you are making a change to the README.md, fixing a typo, or changing code format, please append `[ci skip]` to the end of your commit message, so it won't trigger a new build.
```

Comment:
may as well remove this for now since it isn't implemented into the ci

Reply:
Addressed in `d07c74285af756ad4ed281cbde4a0e9b305d59dc`.